.rat-excludes

```diff
@@ -30,6 +30,9 @@ spark-env.sh.template
 log4j-defaults.properties
 bootstrap-tooltip.js
 jquery-1.11.1.min.js
+d3.min.js
+dagre-d3.min.js
+graphlib-dot.min.js
```

Comment:
Yes good catch - we should do that.

On Tue, May 5, 2015 at 10:14 PM, Shixiong Zhu <notifications@github.com>
wrote:

> In .rat-excludes
> <https://github.com/apache/spark/pull/5729#discussion_r29713742>:
>
> > @@ -30,6 +30,9 @@ spark-env.sh.template
> >  log4j-defaults.properties
> >  bootstrap-tooltip.js
> >  jquery-1.11.1.min.js
> > +d3.min.js
> > +dagre-d3.min.js
> > +graphlib-dot.min.js
>
> Do we need to add dagre-d3 and graphlib-dot to LICENSE? I have already
> added d3 in #5533 <https://github.com/apache/spark/pull/5533>.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/apache/spark/pull/5729/files#r29713742>.
>